contrib/ec2/deis.template.json

```diff
@@ -78,6 +78,13 @@
       "AllowedValues" : [ "PV", "HVM" ],
       "ConstraintDescription" : "must be either PV or HVM"
     },
+    "EC2EBSVolumeType" : {
+      "Description" : "EC2 EBS VolumeType",
+      "Type": "String",
+      "Default": "gp2",
+      "AllowedValues" : [ "gp2", "io1", "standard" ],
```

Comment:
Maybe we simplify this by just hard-coding `gp2`, and if folks want to use provisioned iops they can hack the template with the necessary values?